includes/definitions/discovery/gigabyte-bmc.yaml

```diff
@@ -0,0 +1,40 @@
+mib: GIGABYTE-BMC
+modules:
+    os:
+        hardware: GIGABYTE-BMC::fruPartNumber.0
+        serial: SNMPv2-MIB::snmpSetSerialNo.0
+    processors:
+        data:
+            -
+                oid: processorCollection
```

Comment:
This oid contains processor information not usage data.
Likely you could populate something into inventory with it instead.